third_party/terraform/data_sources/data_source_google_monitoring_uptime_check_ips.go

```diff
@@ -0,0 +1,72 @@
+package google
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceGoogleMonitoringUptimeCheckIps() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleMonitoringUptimeCheckIpsRead,
+
+		Schema: map[string]*schema.Schema{
+			"uptime_check_ips": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"region": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"location": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"ip_address": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func dataSourceGoogleMonitoringUptimeCheckIpsRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
```

Comment:
I wonder whether it actually makes sense to read the project here, or whether we want to just pass a "" into the request, since this call will return an error if the provider project isn't set but the data source doesn't actually need it.

Reply:
Totally right! Project wasn't needed at all.